### DISCUSSION TITLE
Forward Nodes not showing in Grid

### DISCUSSION THREAD

**Discussion by KraigTersigni** [Security-Onion-Solutions/securityonion#3686] [Unsupported Versions] [answered]
I am currently deploying a distributed solution and running into an issue with the forward nodes not showing in the grid. I know they are communicating to the manager because I see them in Fleet / Grafana / and accepted salt-keys. The ManagerSearch is running version 2.30 / I have one forward node at 2.30 and another at 2.40 / both are exhibiting the same behavior. Please see attached to the errors.log file. 
[errors.docx](https://github.com/Security-Onion-Solutions/securityonion/files/6230459/errors.docx)

**Comment by weslambert** [comment, marked as answer]
You might want to try restarting Sensoroni on each forward node, and please attach errors using a plain text file or by formatting them in the comment.

**Reply by KraigTersigni** [reply]
Thank you for your input. We were able to track the problem down to port tcp/5000 being blocked. After opening the port on the manager all is well.
